yamf/router.php

```diff
@@ -18,10 +18,10 @@
 
     $request = find_route($routes, $requestURL);
     if ($request !== null) {
-        require_once 'controllers/' . $request->controller . '.php';
         $controller = new $request->controller;
         $data = $controller->{$request->function}($app, $request);
         if ($data != null) {
+            /** @var View $data */
```

Comment:
`$data` isn't necessarily a `View` -- it could be a `Response`, `JsonResponse`, etc.

Reply:
But it's always `Response`? I can change it to `Response`?

Reply:
Yeah, `Response` should be good.